sdks/python/apache_beam/transforms/core.py

```diff
@@ -1423,15 +1423,27 @@ def _check_fn_use_yield_and_return(fn):
     source_code = _get_function_body_without_inners(fn)
     has_yield = False
     has_return = False
+    return_none_warning = "No iterator is returned by " \
+                          "the process method in DoFn."
```

Comment:
```suggestion
    return_none_warning = ("No iterator is returned by "
                          "the process method in DoFn.")
```

Reply:
This warning could be meaningful if it can mention the faulty DoFn. Can we add the name of the DoFn?

Reply:
Added the class name using `fn.__class__` 👍